### PR TITLE
Added the facter command to check os details 

### PR DESCRIPTION
**Pull request by maheshloni**
As per CSIT community , Need to proper command to fetch the OS details

### CODE REVIEW

File: pkgsrc/dev_compile_script.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2016 Tata Consulting
+# Copyright 2016 Ericsson
 #
```
Comment (pguibert6WIND) [inline pkgsrc/dev_compile_script.sh]: a copyright is never changed
if you want to add Ericsson, please append, but not replace not remove.
thanks

**Review by pguibert6WIND — CHANGES_REQUESTED** [review]
except for the copyright.
I agree with the rest of the commit

**Comment by maheshloni** [comment]
Philippe, 
I have updated your review comments

**Review by maheshloni — COMMENTED** [review]
updated the review comments

**Comment by maheshloni** [comment]
Please merge to master as well as 20170418 and 20170330 release tags

**Review by pguibert6WIND — APPROVED** [review]

**Review by pguibert6WIND — APPROVED** [review]

**Pull request merged**